hw/drivers/lp5523/pkg.yml

```diff
@@ -0,0 +1,34 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+pkg.name: hw/drivers/lp5523
+pkg.description: Driver for the TI LP5523
+pkg.author: "Julian Ingram <julian@apache.org>"
+pkg.homepage: "http://mynewt.apache.org/"
+pkg.keywords:
+    - led
+    - texas
+    - lp5523
+        
+pkg.deps:
+    - hw/sensor
+
```

Comment:
Here you  should also add : 

    - "@apache-mynewt-core/kernel/os"
    - "@apache-mynewt-core/hw/hal"
    - "@apache-mynewt-core/hw/sensor"

pkg.deps.LP5523_CLI:
    - "@apache-mynewt-core/util/parse"